cmd/lifecycle/main.go

```diff
@@ -143,7 +143,9 @@ func verifyReadAccess(imageRef string, keychain authn.Keychain) error {
 		return nil
 	}
 	img, _ := remote.NewImage(imageRef, keychain)
-	if !img.CheckReadAccess() {
+	canRead, err := img.CheckReadAccess()
+	if !canRead {
+		cmd.DefaultLogger.Debugf("Error checking read access: %s", err)
```

Comment:
We should log that without using Debug as some users using Tekton or kpack could not be aware about how to enable log level !!